packages/@react-aria/menu/src/useMenuTrigger.ts

```diff
@@ -79,14 +79,14 @@ export function useMenuTrigger(props: MenuTriggerAriaProps, state: MenuTriggerSt
       id: menuTriggerId,
```

Comment:
Another thing to consider is whether or not we should propagate `isDisabled` through the button/trigger props returned by `useMenuTrigger`/`useComboBox`. This would save us the trouble from having to add these `isDisabled` checks to our press handlers and would save the user from having to pass `isDisabled` to `usePress` + they would automatically get `disabled: true` from the props that `usePress` returns. 

However, it would mean they would need to override that `disabled: true` if they wanted to have a trigger button that is focusable/pressable but has its overlay opening capabilities disabled.

Reply:
Talked with @snowystinger, I'll propagate isDisabled through for useComboBox for now